src/pages/SystemAdmin/SystemAdmin.tsx

```diff
@@ -83,6 +93,23 @@ const SystemAdmin = () => {
           </Tooltip>
         </Grid>
       </Grid>
+      <ModalWrapper
+        open={open}
+        header="Clear All Queues?"
+        onClose={() => setOpen(false)}
+        onCancel={() => setOpen(false)}
+        onSubmit={() => {
+          queueClient.clearQueues()
+          setOpen(false)
+        }}
+        content={
+          <Typography my={2}>
+            All outstanding requests for the system will be deleted. This cannot
```

Comment:
The phrasing here could be a bit confusing because of the way the terms "system" and "request" get used in beergarden.  Also (the original comment I came here to make), this action is kind of misleading because it's actually only going to apply to the local garden, so it might be good to try to clarify that here.  Maybe the message could be something like:

> All outstanding request messages will be deleted for all systems of the local garden. Remote gardens will not be affected.  This action cannot be undone.